src/ape/utils/basemodel.py

```diff
@@ -127,7 +127,6 @@ class Config:
         keep_untouched = (cached_property, singledispatchmethod)
         arbitrary_types_allowed = True
         underscore_attrs_are_private = True
-        anystr_strip_whitespace = True
```

Comment:
this has caused data to get malformed on load if it started or ended with bytes `[9, 10, 11, 12, 13, 32]`, as in `['\t', '\n', '\x0b', '\x0c', '\r', ' ']`

Reply:
No idea why this was there....

Looks like @unparalleled-js might have added in #684?

Reply:
I refactored in that PR and split up Utils.py.

I don't think I added it initially.